src/extensions/surveys/surveys-utils.tsx

```diff
@@ -48,7 +48,7 @@ export const style = (appearance: SurveyAppearance | null) => {
               text-align: left;
               max-width: ${parseInt(appearance?.maxWidth || '300')}px;
               width: 100%;
-              z-index: ${parseInt(appearance?.zIndex || '99999')};
+              z-index: ${parseInt(appearance?.zIndex || '2147482647')};
```

Comment:
lets change it here too https://github.com/PostHog/posthog-js/blob/92ee6c5bad352b8879f44749b7f96a6556ff4a1a/src/extensions/surveys-widget.ts#L40

Reply:
the line above should be using `appearance?.zIndex` as well I guess

Reply:
not really as those are the settings for the widget tab, not the survey customization, and we currently don't allow appearance changes for the tab. we probably can, but for now i'll just leave as const

Reply:
also made the zindex default value a const